model_info/cswiki.damaging.md

```diff
@@ -1,12 +1,12 @@
 Model Information:
 	 - type: GradientBoosting
-	 - version: 0.6.0
```

Comment:
is downgrading the version correct here?

Reply:
Definitely not.  Looking into it.

Reply:
I think I see the issue.  We manually incremented that version number because of some improvements we made to the model.  Now, the default has taken over.  This is some tech debt we forgot about at the time.  I'll make some changes to the Makefile template to allow us to handle this better in the future.